src/test/resources/org/prebid/server/it/openrtb2/ttx/test-auction-ttx-request.json

```diff
@@ -43,7 +43,14 @@
   "ext": {
     "prebid": {
       "bidadjustmentfactors": {
-        "33across": 0.7
+        "mediatypes": {
```

Comment:
Pls change `rubicon` bidder test instead. Other bidder tests should be simple as possible.
Probably we'll need to divide rubicon-tests into some functional way.

Reply:
Moved this logic to multifid rubicon test, because current rubicon-appnexus test, has already to many test conditions